.github/workflows/scripts/get-workflow-run-artifact.js

```diff
@@ -0,0 +1,45 @@
+module.exports = async ({ github, context, core, runId, artifactName }) => {
+    try {
+        const AdmZip = require('adm-zip');
+        const fs = require('fs');
+
+        const { owner, repo } = context.repo;
+        const { data } = await github.rest.actions.listWorkflowRunArtifacts({
+            owner, 
+            repo, 
+            run_id: runId,
+        });
+    
+        const artifact = data.artifacts.find(a => a.name === artifactName);
+        
+        if (!artifact) {
+            throw new Error(`Could not find artifact ${artifactName} in workflow (${runId})`);
+        }
+    
+        const zip = await github.rest.actions.downloadArtifact({
+            owner,
+            repo,
+            artifact_id: artifact.id,
+            archive_format: "zip",
+        });
+    
+        const dir = `./tmp/${artifactName}`;
+        await mkdirRecursive(fs, dir);
```

Comment:
**Suggestion:** maybe use [`fs.mkdirSync()`](https://nodejs.org/api/fs.html#fsmkdirsyncpath-options) instead?

Although it's blocking, I think it shouldn't be too much of an issue here, on the other hand it would make the script simpler. What do you think?